napari/_qt/utils.py

```diff
@@ -323,3 +323,40 @@ def delete_qapp(app):
     # calling a second time is necessary on PySide2...
     # see: https://bugreports.qt.io/browse/PYSIDE-1470
     QApplication.instance()
+
+
+def get_view_direction_in_scene_coordinates(
+    view,
+) -> np.ndarray:
+    """calculate the unit vector pointing in the direction of the view
+
+    Adapted From:
+    https://stackoverflow.com/questions/37877592/
+        get-view-direction-relative-to-scene-in-vispy/37882984
+
+    Parameters
+    ----------
+    view : vispy.scene.widgets.viewbox.ViewBox
+        The vispy view box object to get the view direction from.
+
+    Returns
+    -------
+    view_vector : np.ndarray
+        Unit vector in the direction of the view in scene coordinates
```

Comment:
Looks like order here is zyx, can you just confirm that and add to doc string

Reply:
Done!